lib/parallel.rb

```diff
@@ -215,6 +215,20 @@ def each(array, options={}, &block)
       array
     end
 
+    def any?(array, options={}, &block)
+      block ||= -> (v) { v }
+      !map(array, options.merge(:preserve_results => false)) do |*args|
+        raise Parallel::Kill if block.call(*args)
+      end
+    end
+
+    def all?(array, options={}, &block)
+      block ||= -> (v) { v }
```

Comment:
Ah, the implicit block is the default behavior in Ruby: http://apidock.com/ruby/Enumerable/any%3F

So you can call:

```
[ nil, true, 99 ].any?                            #=> true
```